docs/sources/examples/IRIS_Example.md

```diff
@@ -0,0 +1,50 @@
+# IRIS Example
+
+The following code illustrates the usage of TPOT with the IRIS data set. 
+
+```python
+from tpot import TPOT
+from sklearn.datasets import load_iris
+from sklearn.cross_validation import train_test_split
+
+iris = load_iris()
+X_train, X_test, y_train, y_test = train_test_split(iris.data, iris.target, train_size=0.75)
+
+tpot = TPOT(generations=5)
+tpot.fit(X_train, y_train)
+print(tpot.score(X_train, y_train, X_test, y_test))
+tpot.export('tpot_iris_pipeline.py')
+```
+
+Running this code should discover a pipeline that achieves ~92% testing accuracy, and the corresponding Python code should be exported to the `tpot_iris_pipeline.py` file and look similar to the following:
+
+```python
+import numpy as np
+import pandas as pd
+
+from sklearn.cross_validation import StratifiedShuffleSplit
+from sklearn.svm import SVC
+from sklearn.neighbors import KNeighborsClassifier
+
+# NOTE: Make sure that the class is labeled 'class' in the data file
+tpot_data = pd.read_csv('PATH/TO/DATA/FILE', sep='COLUMN_SEPARATOR')
+training_indeces, testing_indeces = next(iter(StratifiedShuffleSplit(tpot_data['class'].values, n_iter=1, train_size=0.75)))

```

Comment:
Please ensure that `test_size=0.25` is also in this call. See #52 for the reasoning.